### DISCUSSION TITLE
How to aggregate multiple nodes

### DISCUSSION THREAD

**Discussion by tigadad** [freeplane/freeplane#1493] [Help] [answered]
I need to aggregate three nodes onto one. I though I could do this using a free floating node but I can't find how to position the node.

But I still can't aggregate the nodes.
![screen](https://github.com/freeplane/freeplane/assets/109116597/9d8ad9f1-c646-42ad-944f-33a9ded2abc1)

**Comment by HaaiHenkie** [comment, marked as answer]
Hello!

I think you would like to create a summary node.

1. Select the 3 nodes
2. From the menu choose: Insert > New node > New summary node

Success!

**Reply by tigadad** [reply]
Brilliant. Thank you.
